draft-ietf-lake-authz.md

```diff
@@ -632,6 +641,196 @@ If present, V decrypts and verifies opaque_state as received from W. If that ver
+
+This is different from the EDHOC reverse message flow defined in {{Appendix A.2.2 of RFC9528}}, since we make no assumption about whether U or V is a CoAP server.
+
+### U is the Initiator {#u-initiator}
+
+For clarity, we first present the default scenario with U as Initiator, as described in {{protocol-overview}} and {{U-V}}.
+Note that Voucher_Info and Voucher are carried in EDHOC message_1 and message_2, respectively.
+
+~~~~~~~~~~~ aasvg
++-----+-----+                  +-----------+
+|     U     |                  |     V     |
+| Initiator |                  | Responder |
++-----+-----+                  +-----+-----+
+      |                              |
+      |                              |
+      |       EDHOC message_1        |                              W
+      +----------------------------->|                              |
+      | EAD_1 = (TBD1, Voucher_info) |                              |
+      |                              |             VREQ             |
+      |                              +----------------------------->|
+      |                              | (SS, G_X, Voucher_Info,      |
+      |                              |  H_message_1, ?opaque_state) |
+      |                              |                              |
+      |                              |             VRES             |
+      |                              +<---------------------------->|
+      |                              |    (Voucher, ?opaque_state)
+      |       EDHOC message_2        |
+      +<-----------------------------|
+      |   EAD_2 = (TBD2, Voucher)    |
+      |                              |
+      |       EDHOC message_3        |
+      +----------------------------->|
+      |                              |
+      |                              |
+~~~~~~~~~~~
+{: #fig-u-initiator title="ELA when U is the EDHOC Initiator." artwork-align="center"}
+
+### U is the Responder {#u-responder}
+
+ELA also works with U as the EDHOC Responder, a setup we refer to as the ELA reverse flow, as shown in {{fig-u-responder}}.
+
+We present this variant as a set of changes to the regular protocol flow, so that U can run as the EDHOC Responder.
+The changes include:
+
+* Voucher_Info and Voucher are transported in EDHOC message_2 and message_3, respectively (instead of message_1 and message_2).
+* The EAD_2 and EAD_3 fields carry EAD items identified with labels TBD1 and TBD2, respectively.
+* The VREQ / VRES protocol takes place between message_2 and message_3.
+* The Voucher_Request carries G_Y instead of G_X, and the transcript hash TH_2 instead of the hash H_message_1.
+
+~~~~~~~~~~~ aasvg
++-----+-----+                  +-----------+
+|     U     |                  |     V     |
+| Initiator |                  | Responder |
++-----+-----+                  +-----+-----+
+      |                              |
+      |       Trigger Message        |
```

Comment:
Given the trigger message, put the voucher_info in it.

Reply:
The thing is that certain network technologies are (1) device sends first and others are (2) gateway sends first. That's why the trigger is optional. 
The case for U as Responder addresses the case when gateway sends first.